lib/digital_signature_api/crl/date_utils.ex

```diff
@@ -0,0 +1,38 @@
+defmodule DigitalSignature.DateUtils do
+  @moduledoc false
+
+  defmacro __using__(_) do
```

Comment:
You can just define function `convert_date` and than `import DateUtils` to call function without module name